servers/physics_3d/shape_3d_sw.h

```diff
@@ -390,20 +390,36 @@ struct ConcavePolygonShape3DSW : public ConcaveShape3DSW {
 
 struct HeightMapShape3DSW : public ConcaveShape3DSW {
 	Vector<real_t> heights;
```

Comment:
Makes sense! In order to keep things simple for now I can probably just stick to float instead of real_t, and in the future more formats can possibly be added.